### ISSUE TITLE
[UX] Clarify/unify account status terminology across admin pages

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#1249]
There is nothing in the UI about how to approve user accounts. There's no bulk operation, no checkbox, no listing. What is actually happening is that these accounts are blocked. Can we some way communicate to users that in order to "approve" accounts what they actually need to do is make them "Active" instead of "Blocked"?

**Comment by klonos** [comment]
I don't get it. ...isn't the user status by default set to active anyways?

**Comment by docwilmot** [comment]
Off topic-ish, but you know what would be nice? If we had online help pages baked into Backdrop's menu bar, like context-aware links to api.backdrop cms.org and the user help pages (when those become available). So you're on a user-related page and you hover over the help button and get links related to account stuff. Just a thought. Think this is worth an issue?

**Comment by klonos** [comment]
> Think this is worth an issue?

Yep.

**Comment by jenlampton** [comment]
> isn't the user status by default set to active anyways?

No, by default Visitors can create accounts `but administrator approval is required `

**Comment by klonos** [comment]
Good use case for a dashboard block/widget over in #495 I guess (users pending approval).

**Comment by klonos** [comment]
...turns out @docwilmot has already included that in his list.

**Comment by jenlampton** [comment]
What good does a list of users pending approval do us, if there's no way to `Approve` them?  All we can do in the current UI is `Unblock` them.

**Comment by klonos** [comment]
> What good does a list of users pending approval do us, if there's no way to ```Approve``` them? All we can do in the current UI is ```Unblock``` them.

I think that's the thing. In Drupal (and uninhibitedly in Backdrop), in the context of the user approval procedure we ```not-approved-yet``` = ```blocked```.

**Comment by Dinornis** [comment]
The `Manage user accounts` page uses the terms:
**Status:** `Active` | `Blocked`
**Operations:** `Block user(s)` | `Unblock user(s)`

Perhaps `Activate user(s)` would be better choice?

The `Who can register accounts?` choice on the `Account settings` page has an option:
`Visitors, but administrator approval is required`
could be changed to:
`Visitors, but administrator needs to activate the account on the _Manage user accounts page_ (link) or when editing the individual account.`
